conf/machine/uz7ev-evcc-hdmi-v-n.conf

```diff
@@ -0,0 +1,48 @@
+#@TYPE: Machine
+#@NAME: uz7ev-evcc-hdmi-v-n
+#@DESCRIPTION: Machine support for UltraZed-EV with EVCC, including HDMI and VCU support.
+#
+
+SOC_VARIANT = "ev"
+
+MACHINEOVERRIDES = "uz:uz7ev:uz7ev-evcc:uz7ev-evcc-hdmi:${MACHINE}"
```

Comment:
This machine should be an override to uz7ev-evcc-hdmi-v as well
so MACHINEOVERRIDES = "uz:uz7ev:uz7ev-evcc:uz7ev-evcc-hdmi:uz7ev-evcc-hdmi-v:${MACHINE}"